product/models.py

```diff
@@ -0,0 +1,98 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'categories'
+
+class SubCategory(models.Model):
+    name     = models.CharField(max_length=30)
+    category = models.ForeignKey('Category', on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'sub_categories'
+
+class Product(models.Model):
+    sub_category    = models.ForeignKey('SubCategory', on_delete=models.PROTECT)
+    name            = models.CharField(max_length=45)
+    description     = models.CharField(max_length=60)
+    price           = models.IntegerField()
+    image_url       = models.CharField(max_length=3000)
+    sales_unit      = models.CharField(max_length=30, null=True)
+    amount          = models.CharField(max_length=30, null=True) # 중량/용량
+    origin          = models.CharField(max_length=30, null=True)
+    storage_method  = models.ForeignKey('StorageMethod', on_delete=models.SET_NULL, null=True) #DO_NOTHING ? 
+    expiration_date = models.CharField(max_length=100, null=True) # 정해진 날짜가 없고 대부분 짧은 텍스트로 설명이 되어있음
+    stock           = models.IntegerField()
```

Comment:
```suggestion
    stock           = models.PositiveIntegerField(default=0)
```
재고수량은  음수로 내려가진 않을테니 PositiveIntegerField는 어떨까요? 인티저 필드를 지정할때는 기본값을 주도록 해주세요.

Reply:
네 수정하겠습니다~!